packages/nns/src/canisters/governance/request.converters.ts

```diff
@@ -899,15 +900,14 @@ export const toDisburseNeuronRequest = ({
   amount,
 }: {
   neuronId: NeuronId;
-  toAccountId?: string;
+  toAccountId?: AccountIdentifierClass;
```

Comment:
would `toAccountIdentifier` or `toAccountIdentifierHash` be more accurate here?

Reply:
sure, `toAccountIdentifier` makes sens

Reply:
done